WooCommerce/WooCommerceTests/ViewRelated/Coupons/CouponListViewModelTests.swift

```diff
@@ -193,6 +206,19 @@ final class CouponListViewModelTests: XCTestCase {
     }
 
     func test_shouldDisplayFeedbackBanner_returns_false_if_state_is_empty() {
+        // Given
+        let stores = MockStoresManager(sessionManager: .makeForTesting())
+        stores.whenReceivingAction(ofType: SettingAction.self) { action in
+            switch action {
+            case let .retrieveCouponSetting(_, onCompletion):
+                onCompletion(.success(true))
+            default:
+                break
+            }
+        }
```

Comment:
nit: is this setup still needed since a successful empty result doesn't check the coupons setting anymore?

Reply:
Thanks for noticing! Removed the redundant setup in bca8981bf8d78e658d69a9010135b07d1235ff81.